### ISSUE TITLE
Checkpoint selection

### ISSUE THREAD

**Issue by orspiegel** [neurdylab/UnEBOLT#1]
Hello! Thank you for the great paper and for making the code and data available.

I am trying to reproduce the results reported for unseen-subject prediction (P=64), but I have not been able to match the numbers in the paper using the provided training/inference pipeline, dataset, and split.

Could you please clarify which checkpoint was used for the reported results? Was it the final checkpoint after 20 epochs, or was the checkpoint selected according to a validation metric (e.g. validation loss, Tcorr, Pixcorr, etc.)?
If checkpoint selection was used, it would also be very helpful to know the exact selection criterion.

<img width="1272" height="142" alt="Image" src="https://github.com/user-attachments/assets/31864a2c-6915-41f6-812a-72389b8f16e1" />
Thanks a lot!
